examples/nextjs-with-supabase-auth/package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "with-supabase-auth",
-  "version": "1.0.0",
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start"
-  },
-  "dependencies": {
-    "@supabase/supabase-js": "^1.2.1",
-    "@supabase/ui": "^0.6.1",
-    "next": "latest",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "swr": "0.2.3"
-  },
-  "license": "MIT"
-}
+    "name": "with-supabase-auth",
+    "version": "1.0.0",
+    "scripts": {
+        "dev": "next dev",
+        "build": "next build",
+        "start": "next start"
+    },
+    "dependencies": {
+        "@supabase/supabase-js": "^1.11.9",
+        "@supabase/ui": "^0.24.2",
+        "next": "latest",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
+        "swr": "^0.5.5"
```

Comment:
just to check - did you run the example to ensure all these versions are working?